web/cypress/integration/countrypanel.spec.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable no-undef */
```

Comment:
@madsnedergaard any tips for how to use the cypress keywords without disabling eslint rules?

Reply:
We can either add the keywords to GLOBALS in eslint config, but we can also just add this plugin :)
https://www.npmjs.com/package/eslint-plugin-cypress